libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -720,7 +720,7 @@ interface InputType {
     LnUrlPay(LnUrlPayRequestData data);
     LnUrlWithdraw(LnUrlWithdrawRequestData data);
     LnUrlAuth(LnUrlAuthRequestData data);
-    LnUrlError(LnUrlErrorData data);
+    LnUrlEndpointError(LnUrlErrorData data);
```

Comment:
Do we have to change the naming here? It is better to leave the current API unmodified in this refactoring.

Reply:
Done: fcebd47fd8bfb369d9d829cd0d42c6548b214390

It was needed due to a FRB conflict with the identically named struct, but that was fixed.